R/prep_green_brown_bars.R

```diff
@@ -1,6 +1,100 @@
-prep_green_brown_bars <- function(results_portfolio = NULL) {
-  if (is.null(results_portfolio)) {
-    data_out <- use_toy_data("green_brown_bars")
+#' Prepare data input for plotting green brown bar chart
+#'
+#' Prepare data input for plotting green brown bar chart based on PACTA for
+#' investors output files. These files must have been wrangled with
+#' `prep_data_executive_summary()` before they can be passed to this function.
+#'
+#' @param results_portfolio Data frame that contains pre-wrangled portfolio
+#'   level PACTA results from a PACTA for investors analysis.
+#' @param scenario_selected Character. Must be a `scenario` featured in the
+#'   `scenario_thresholds` data set.
+#'
+#' @return data.frame
+prep_green_brown_bars <- function(results_portfolio,
+                                  scenario_selected) {
```

Comment:
What is this argument about exactly? Isn't it fixed in case of ES what kind of scenario sources we're using?

Reply:
This picks one of the three scenarios that we can choose from given the scenario_source. We could set a default here, since we are using GECO2021 across the ES anyway. So it would work to just use the default 1.5° scenario. But maybe also leave the option to adjust, so that we can extend the ES to use IEA/WEO scenarios easily?

Reply:
Let's set a default. It will make the code easier to use, especially for users who are not aware that the decision is to use GECO2021 for ES. This way we don't force anyone to make the choice but leave it open if one wants to choose.

Reply:
done